packages/quantic/cypress/integration/breadcrumb-manager/breadcrumb-manager.cypress.ts

```diff
@@ -94,11 +96,14 @@ describe('quantic-breadcrumb-manager', () => {
 
         Expect.dateFacetBreadcrumb.displayBreadcrumb(true);
         Expect.dateFacetBreadcrumb.displayLabel(true);
-        Expect.dateFacetBreadcrumb.labelContains('Date');
+        Expect.dateFacetBreadcrumb.labelContains(dateFacetName);
         Expect.dateFacetBreadcrumb.numberOfValues(1);
         Expect.dateFacetBreadcrumb.firstBreadcrumbValueLabelContains(
           timeframeLabel1
         );
+        Expect.dateFacetBreadcrumb.firstBreadcrumbAltTextContains(
+          `${dateFacetName} ${clearActionName}`
```

Comment:
Well I think so 😂 however, the reader reads the actual text in order of it being present in the DOM.

And since the rendered HTML for each breadcrumb values is like this:

![image](https://user-images.githubusercontent.com/2488155/180071010-ddccf945-5474-4713-b3b8-b773ea671735.png)

the reader actually reads the visible text before reading the rest of the assistive text, so it always starts with the value name.
I think it makes sense too, if you are tabbing through those, you kinda know where you are so you can more easily just listen to the value to clear first instead of listening to "clear ..." "clear..." "clear ..." every time.
That was my logic anyway.